user-interface/3001-VOTE-vote.md

```diff
@@ -1 +1,157 @@
+
+for each proposal:
+
+- **must** see the type of proposal (<a name="3001-VOTE-007" href="#3001-VOTE-007">3001-VOTE-007</a>)
+- **must** see the proposal title (<a name="3001-VOTE-097" href="#3001-VOTE-097">3001-VOTE-097</a>)
+- **should** see a summary of what the type of proposed change is, without looking at details (network, new market etc)
+  - for network parameters: **should** see what parameter is being changed and new value
+  - for network parameters: **could** see what the current values are for that parameter
+  - for network parameters: **could** see if there are other open proposals for the same parameter
+  - for new markets: **should** see the type of market (e.g. Future)
+  - for new markets: **could** see the type trading mode of the market (e.g. auction, continuous)
+  - for new markets: **should** see the name of the new market
+  - for new markets: **should** see the code of the new market
+  - for new markets: **should** see the settlement asset of the new market (not just asset ID but asset Symbol)
+  - for new markets: **could** see a summary of the oracle used for settlement
+  - for market changes: **should** see the name of the market being changed
+  - for market changes: **should** see a summary of what parameters are being changed
+  - for market changes: **should** see a the proposed values for parameters
+  - for market changes: **should** see a the current values for that parameter
+  - for market changes: **could** see if there are other open proposals for the same market
+  - for new assets: **must** see the name of the new asset (<a name="3001-VOTE-026" href="#3001-VOTE-026">3001-VOTE-026</a>)
+  - for new assets: **must** see the code of the new asset (<a name="3001-VOTE-027" href="#3001-VOTE-027">3001-VOTE-027</a>)
+  - for new assets: **must** see the source of the new asset (e.g. ERC20) (<a name="3001-VOTE-028" href="#3001-VOTE-028">3001-VOTE-028</a>)
+  - for new assets (if source is ERC20): **must** see contract address (<a name="3001-VOTE-095" href="#3001-VOTE-095">3001-VOTE-095</a>)
+  - for new assets (if source is ERC20): **must** see if the Asset has been whitelisted on the bridge (<a name="3001-VOTE-096" href="#3001-VOTE-096">3001-VOTE-096</a>)
+  - for asset changes: **must** see name of asset being changed (<a name="3001-VOTE-029" href="#3001-VOTE-029">3001-VOTE-029</a>)
+  - for asset changes: **must** see the parameter(s) being changed (<a name="3001-VOTE-030" href="#3001-VOTE-030">3001-VOTE-030</a>)
+  - for asset changes; **must** see the new value for the parameters being changed (<a name="3001-VOTE-031" href="#3001-VOTE-031">3001-VOTE-031</a>)
+  - for asset changes: **could** see if there are other open proposals for the same parameter(s)
+  - for asset changes: **should** see the current values for these parameters
+  - for freeform: **must** see a summary of the proposal (suggest the first x characters of the proposal blob)
```

Comment:
can we add could format these as markdown?

Reply:
link 106 mentioned markdown, it's a should at the moment (because it wasn't implemented at the time). happy to upgrade it?

Reply:
upgraded it to a must because I think it is and Dex said he was creating a ticket to make it so https://github.com/vegaprotocol/specs/pull/1302/commits/f533e91c1b8bb0cba43a0e7604376a9d0d37d36f